test/helpers/e2e.go

```diff
@@ -303,7 +320,7 @@ func newSimulatedEVML2SovereignChain(t *testing.T) (
 	genesisAllocMap := map[common.Address]types.Account{
 		l2BridgeProxyAddr: {Balance: premineBalance},
 	}
-	client, setup := NewSimulatedBackend(t, genesisAllocMap, deployerAuth)
+	client, setup := NewSimulatedBackend(t, genesisAllocMap, deployerAuth, aggOracleCommitteeConfig.EnableAggOracleCommittee)
```

Comment:
Consider wrapping the line, because linter complains.
```suggestion
	client, setup := NewSimulatedBackend(t, genesisAllocMap, deployerAuth, 
    aggOracleCommitteeConfig.EnableAggOracleCommittee)
```

Reply:
done